recipes-temporary-patches/boot2qt/connman_%.bbappend

```diff
@@ -0,0 +1,5 @@
+
+FILESEXTRAPATHS_prepend := "${BSPDIR}/sources/meta-boot2qt/meta-boot2qt-distro/recipes-connectivity/connman/files:"
+
+SRC_URI += "file://0001-disable-connman-when-booting-over-nfs-1.34.patch"
```

Comment:
I assume using += will just append this file and the 1.35.patch will still be there, so it will try to find it. Have you tested this that it works like this?

Reply:
Yeah, use `_append`

Reply:
I meant to say that SRC_URI will contain both 
1.35 and 1.34 patch and might fail again when trying to look for 1.35. So just test locally that it actually works

Reply:
Yeah, that was the case. It was still looking for 1.35